code/datums/status_effects/debuffs.dm

```diff
@@ -824,3 +824,79 @@
 
 /datum/status_effect/transient/drugged/on_remove()
 	owner.update_druggy_effects()
+
+#define FAKE_COLD 1
+#define FAKE_FOOD_POISONING 2
+#define FAKE_RETRO_VIRUS 3
+#define FAKE_TURBERCULOSIS 4
+
+/datum/status_effect/fake_virus
+	id = "fake_virus"
+	duration = 3 MINUTES
+	status_type = STATUS_EFFECT_REPLACE
+	tick_interval = 2
+	alert_type = null
+	/// So you dont get the most intense messages immediately
+	var/msg_stage = 0
+	/// Which disease we are going to fake?
+	var/current_fake_disease
+
+/datum/status_effect/fake_virus/on_creation()
+	current_fake_disease = pick(FAKE_COLD, FAKE_FOOD_POISONING, FAKE_RETRO_VIRUS, FAKE_TURBERCULOSIS)
+	. = ..()
+
+/datum/status_effect/fake_virus/tick()
```

Comment:
shouldn't be checking which `current_fake_disease` it is each tick. Instead you should have the lists of fake_emotes + fake_msgs available already in a list once `on_creation()` is done, this should significantly cut down on the amount of checks that need to be done.

Reply:
Like this?